10_simplification/main.go

```diff
@@ -145,31 +147,122 @@ can be taken.
 
 func main() {
 	// TODO: declare a variable `x` of type `int` and set it to 10
+	// x := 10
 	// TODO: declare a variable `y` of type `int` and set it to 20
+	// y := 20
 	// TODO: declare a variable `name` of type `string` and set it to "John"
+	// name := "John"
 	// TODO: declare a variable `z` of type `int` without assigning a value
+	// var z int
 	// TODO: assign the sum of `x` and `y` to `z`
-
+	// z = x + y
 	// TODO: write a loop that prints all numbers from 0 to 10
+	for i := 0; i <= 10; i++ {
+		fmt.Println(i)
+	}
 	// TODO: write a loop that prints all numbers from 10 to 0
+	for j := 10; j > 0; j-- {
+		fmt.Println(j)
+	}
 	// TODO: write a loop that prints all even numbers from 0 to 10
+	for k := 0; k < 10; k++ {
+		if k%2 == 0 {
+			fmt.Println(k)
+		}
+	}
 	// TODO: write a loop that prints the first 10 odd numbers
+	for l := 0; l < 10; l++ {
+		if l%2 != 0 {
+			fmt.Println(l)
+		}
+	}
```

Comment:
This prints the first odd numbers under 10, the task is to print the first 10 odd numbers.

I.e., you print 1, 3, 5, 7, 9
You need to print 1, 3, 7, 9, 11, 13, 15, 17, 19, 21

Reply:
Done!